avocado/core/nrunner.py

```diff
@@ -824,32 +824,32 @@ def __init__(self, echo=print, prog=None, description=None):
             prog = self.PROG_NAME
         if description is None:
             description = self.PROG_DESCRIPTION
+        self._class_commands_method = self._get_commands_method()
         self._setup_parser(prog, description)
 
     def _setup_parser(self, prog, description):
         self.parser = argparse.ArgumentParser(prog=prog,
                                               description=description)
         subcommands = self.parser.add_subparsers(dest='subcommand')
         subcommands.required = True
-        for cmd_meth in self._get_commands_method_without_prefix():
-            attr = "CMD_%s_ARGS" % cmd_meth.upper()
-            cmd = cmd_meth.replace('_', '-')
+        for cmd, meth in self._class_commands_method.items():
+            attr = "CMD_%s_ARGS" % cmd.upper().replace('-', '_')
             cmd_parser = subcommands.add_parser(
                 cmd,
-                help=self._get_command_method_help_message(cmd_meth))
+                help=self._get_command_method_help_message(meth))
```

Comment:
Maybe a more explicit name `method`?

Reply:
Thanks @beraldoleal, I have updated this and a few other variables in this method.